frontend/src/scenes/dashboard/EmptyDashboardComponent.tsx

```diff
@@ -92,15 +65,19 @@ export function EmptyDashboardComponent(): JSX.Element {
                     <SkeletonOne />
                 </Col>
                 <Col span={24} lg={12}>
-                    <SkeletonTwo />
+                    <Card>
+                        <SkeletonGraph />
+                    </Card>
                 </Col>
             </Row>
             <Row gutter={16} className="fade-out-graphs">
                 <Col span={24} lg={12}>
                     <SkeletonOne />
                 </Col>
                 <Col span={24} lg={12}>
-                    <SkeletonTwo />
+                    <Card>
+                        <SkeletonGraph />
```

Comment:
`SkeletonGraph` is nice but I think `SkeletonTwo` was a better fit here due to sizing: `SkeletonGraph` is unusually tall:
| `SkeletonTwo` | `SkeletonGraph` 
| --- | --- |
| <img width="1389" alt="Screen Shot 2021-12-01 at 10 36 54" src="https://user-images.githubusercontent.com/4550621/144209940-407cb1fa-84c1-4972-b770-a932f9358a02.png"> | <img width="1387" alt="Screen Shot 2021-12-01 at 10 37 39" src="https://user-images.githubusercontent.com/4550621/144209936-b4bfb2e3-a242-4f1a-b9e5-69307663c0bc.png"> |